runtime/oti/VMHelpers.hpp

```diff
@@ -2057,13 +2057,20 @@ class VM_VMHelpers
 		return rc;
 	}
 
+#if JAVA_SPEC_VERSION >= 19
+	static VMINLINE bool isMounted(J9VMContinuation *continuation)
+	{
+		return (NULL != continuation->carrierThread);
+	}
+#endif /* JAVA_SPEC_VERSION >= 19 */
+
 	static VMINLINE bool
-	needScanStacksForContinuation(J9VMThread *vmThread, j9object_t continuationObject) {
+	needScanStacksForContinuation(J9VMThread *vmThread, j9object_t continuationObject, bool scanOnlyUnmounted = false) {
```

Comment:
To match the existing format in the file.
```suggestion
	needScanStacksForContinuation(J9VMThread *vmThread, j9object_t continuationObject, bool scanOnlyUnmounted = false)
	{
```

Reply:
Also, missing a function description (Doxygen format). See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-function-parameters-clearly.